src/test/java/ir/sahab/uncaughtexceptionrule/UncaughtExceptionExtensionTest.java

```diff
@@ -0,0 +1,54 @@
+package ir.sahab.uncaughtexceptionrule;
+
+import org.junit.jupiter.api.*;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@ExtendWith(UncaughtExceptionExtension.class)
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+class UncaughtExceptionExtensionTest {
+
+    @Test
+    @Order(1)
+    public void testAAShouldPasse() {
+        assertTrue(true);
+    }
+
+    // It is a test to test failure of test. So it is disabled.
+    // If you are in doubt, you can enable it!
+    @Test
+    @Order(2)
```

Comment:
Why you did order the unit tests? If you have no reason for that then remove ordering.